File: backend/ftp.py
from ftplib import FTP
import xml.etree.ElementTree as ET

target = "IDV10751.xml"

ftp = FTP('ftp.bom.gov.au');
ftp.login()
ftp.cwd('anon/gen/fwo/');
gFile = open(target, 'wb')
ftp.retrbinary('RETR ' + target, gFile.write)
gFile.close()
ftp.quit()

File: backend/ftp.php
<?php 
$target = "IDV10751.xml";
$url = "ftp.bom.gov.au";

// connect to the server
$conn = ftp_connect($url) or die("Could not connect to server");
$username = "anonymous";
$password = "guest";
// login as anonymous
if (ftp_login($conn, $username, $password)) {
    echo "Connection established";
}
else {
    echo "Could not establish connection";
}

// close the connection
ftp_close($conn);

?>
